README.md

```diff
@@ -350,9 +350,13 @@ Here, we've got our `movies` as an object in props. We also have our `params`
 object which was returned from `useParams` based on the current URL. In this
```

Comment:
In line 349 it should say "...as an array in props."

Reply:
Fixed!